pkg/api/domains/domains.go

```diff
@@ -179,9 +179,15 @@ func ReturnSingleDomain(w http.ResponseWriter, r *http.Request) {
 
 	glog.V(4).Info("GET /domains/" + key)
 
+	var found bool
 	for _, domains := range Domains {
 		if domains.ID == key {
+			found = true
```

Comment:
you should get rid of found, and just return here.

Reply:
I'll take your word for it.  I've always thought and had been taught that, at least in Java context, that was bad practice.

Reply:
right but go is different.  Happy path is return right away when you can.